component/tests/vshn.yml

```diff
@@ -4,9 +4,13 @@ parameters:
       - type: https
         source: https://raw.githubusercontent.com/projectsyn/component-crossplane/v2.3.0/lib/crossplane.libsonnet
         output_path: vendor/lib/crossplane.libsonnet
+      - type: https
+        source: https://raw.githubusercontent.com/appuio/component-openshift4-operators/v1.4.0/lib/openshift4-operators.libsonnet
+        output_path: vendor/lib/openshift4-operators.libsonnet
 
   facts:
     cloud: cloudscale
+    distribution: openshift4
```

Comment:
This will probably break a lot of things when we want to apply to kindev. Please don't add openshift specific things here.

Reply:
Removed it again